woocommerce/woocommerce-gutenberg-products-block
# Fix Actions and Filters documentation with broken links #5808

Fixes #5808

How to test the changes in this Pull Request:

1. npm run build:docs
2. Open docs/extensibility/actions.md
3. Confirm relative Source link paths are correct.
4. Example: [StoreApi/Utilities/CartController.php](../../src/StoreApi/Utilities/CartController.php)
5. Open docs/extensibility/filters.md
6. Confirm relative Source link paths are correct.
7. Example: [BlockTypesController.php](../../src/BlockTypesController.php)
